src/main/java/se/simonsoft/cms/transform/service/TransformServiceXsl.java

```diff
@@ -326,6 +329,16 @@ private void addFolderExists(CmsPatchset patchset, CmsItemPath parentPath) {
 		}
 	}
 	
+	private String decodeHref(String href) {
+		try {
+			href = java.net.URLDecoder.decode(href, "UTF-8");
+		} catch (UnsupportedEncodingException e) {
+			e.printStackTrace();
+			throw new IllegalArgumentException("Could not decode file name: " + href);
```

Comment:
Could not decode URL:

Reply:
Sort out logging. Same message in log and exception. 

Never use e.printStackTrace(); Barbaric.